packages/frontend-api/src/Resources/config/graphql-types/Query/QueryDecorator.types.yaml

```diff
@@ -378,6 +378,10 @@ QueryDecorator:
                         type: "String"
                         defaultValue: null
                         description: "Search for city or post code"
+                    coordinates:
+                        type: "Coordinates"
+                        defaultValue: null
+                        description: "Returns stores with distance from given coordinates"
```

Comment:
```suggestion
                        description: "Returns stores with distance from the store's coordinates"
```

Reply:
I don't think your suggestion is right here. I am describing the input variable that accepts coordinates from user and based on those coordinates returns stores.

Reply:
ah, true :+1: